maintenance_plan/models/maintenance.py

```diff
@@ -0,0 +1,221 @@
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
+from datetime import timedelta
```

Comment:
Could you order the imports by following https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports ?